### PR TITLE
Various embed-related sending stuff

### PR DESCRIPTION
**Pull request by Sorunome**
handles #520 and #517

### CODE REVIEW

**Comment by Sorunome** [comment]
Soru just realised this won't send properly for bot puppets

**Comment by Sorunome** [comment]
There we go, now it does that, too

File: src/bot.ts [hunk 1]
```diff
@@ -414,18 +416,49 @@ export class DiscordBot {
         }
         try {
             this.channelLock.set(chan.id);
-            if (!botUser) {
-                // NOTE: Don't send replies to discord if we are a puppet.
+            if (!roomLookup.canSendEmbeds) {
+                // NOTE: Don't send replies to discord if we are a puppet user.
```
Comment (Half-Shot) [inline src/bot.ts]: I've been bitten by this in #522 and we should fix it.

Comment (Sorunome) [reply src/bot.ts]: still, bot puppets can send embeds just fine, whichlook more nicely. So `!roomLookup.canSendEmbeds` would have a different fallback

**Comment by Sorunome** [comment]
Handles #522 now, too

**Pull request merged**